posthog/hogql_queries/sessions_timeline_query_runner.py

```diff
@@ -0,0 +1,191 @@
+from datetime import timedelta
+import json
+from typing import Dict, Optional, Any, cast
+from posthog.api.element import ElementSerializer
+
+
+from posthog.clickhouse.client.connection import Workload
+from posthog.hogql import ast
+from posthog.hogql.parser import parse_select
+from posthog.hogql.query import execute_hogql_query
+from posthog.hogql.timings import HogQLTimings
+from posthog.hogql_queries.query_runner import QueryRunner
+from posthog.models import Team
+from posthog.models.element.element import chain_to_elements
+from posthog.schema import EventType, SessionsTimelineQuery, SessionsTimelineQueryResponse, TimelineEntry
+
+
+class SessionsTimelineQueryRunner(QueryRunner):
```

Comment:
All query runners will _eventually_ need to implement the `to_persons_query` - but think it's fine to leave this out for now

Reply:
That's good to know. Will skip for now

Reply:
The only thing this needs to work automatically is to return a subquery with one column: `person_id`. Soon possibly also breakdown columns, but that's not here yet. So this might even be all you need:

```python
def persons_query(self)
    return parse_select(`select distinct person_id from {query}`, {'query': self.query()})
```

Reply:
Hmm, yeah, why not, added basically this. Though looks like there's no way to `.calculate()` the persons yet, so the query runner pattern doesn't make `to_persons_query()` testable

Reply:
For now I kept `_toInt64` and made is to that close function name match suggestions don't include functions prefixed with the underscore.
Ideally the type system would know that the `-OrNull` variant can't work here (the identifier referring to a `DateTime64` column) and would use the direct one – but we'll need to pair on this, as the current system is a beast I don't fully get 😄 @mariusandra